jujugui/static/gui/src/app/init/changes-utils.js

```diff
@@ -1,62 +1,40 @@
-/*
-This file is part of the Juju GUI, which lets users view and manage Juju
-environments within a graphical interface (https://launchpad.net/juju-gui).
-Copyright (C) 2012-2013 Canonical Ltd.
-
-This program is free software: you can redistribute it and/or modify it under
-the terms of the GNU Affero General Public License version 3, as published by
-the Free Software Foundation.
-
-This program is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
-SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
-General Public License for more details.
-
-You should have received a copy of the GNU Affero General Public License along
-with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
+/* Copyright (C) 2017 Canonical Ltd. */
 'use strict';
 
-YUI.add('changes-utils', function(Y) {
-
-  var ChangesUtils = {};
-
-  var removeBrackets = /^\(?(.{0,}?)\)?$/;
+class ChangesUtils {
+  constructor() {
+    this.removeBrackets = /^\(?(.{0,}?)\)?$/;
```

Comment:
I think I'd have this as a constant defined outside the class....unless for some reason you need this accessible on the instance itself?